lib/sedna/core/joint_inference/joint_inference.py

```diff
@@ -32,13 +32,22 @@ class TSBigModelService(JobBase):
     Provides RESTful interfaces for large-model inference.
     """
 
-    def __init__(self, estimator=None, config=None):
-        super(TSBigModelService, self).__init__(
-            estimator=estimator, config=config)
+    def __init__(self, estimator=None):
+        """
+        Initial a big model service for JointInference
+        :param estimator: Customize estimator
+        """
+
+        super(TSBigModelService, self).__init__(estimator=estimator)
```

Comment:
If this lines is not related to the bug, you should move them to another commit.

Reply:
fix